.gitignore

```diff
@@ -0,0 +1 @@
+/my-project
```

Comment:
I don't think we need this file here

Reply:
True, though I found it convenient to make sure _I_ didn't inadvertently check in `my-project`.